controllers/rabbitmq/rabbitmq_controller.go

```diff
@@ -356,6 +363,31 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ct
 
 	if clusterReady {
 		instance.Status.Conditions.MarkTrue(condition.DeploymentReadyCondition, condition.DeploymentReadyMessage)
+		// Let's wait DeploymentReadyCondition=True to apply the policy
+		queue := instance.Spec.QueueType
+		ismirrored := isMirroredPolicyApplied(ctx, helper, instance, r.config)
```

Comment:
Do I get that right that with the default ti Mirrored (what we want), every reconciliation will log a new line with the ha-policy of the cluster? If so, does that mean if will also perform one call into the container for each reconciliation?